client/src/partials/users/userPermission.modal.html

```diff
@@ -0,0 +1,41 @@
+<form name="PermissionForm" bh-submit="UserPermissionModalCtrl.submit()">
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="title">
+        {{ "USERS.PERMISSIONS" | translate }}
+        <label class="label label-warning">{{ "USERS.PERMISSIONS" | translate }}</label>
```

Comment:
Just FYI, there is also a class ["badge badge-warning"](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/less/bhima-bootstrap.less#L144) created specifically for this usage. The "label" class is often too small and not properly centered.

It was developed in response to [#263](https://github.com/IMA-WorldHealth/bhima-2.X/issues/263).